plugin/storage/integration/integration.go

```diff
@@ -121,6 +125,52 @@ func (s *StorageIntegration) waitForCondition(t *testing.T, predicate func(t *te
 	return predicate(t)
 }
 
+func (s *StorageIntegration) InitArchiveStorage(t *testing.T, storageFactory storage.Factory, logger *zap.Logger) bool {
+	archiveFactory, ok := storageFactory.(storage.ArchiveFactory)
+	if !ok {
+		logger.Info("Archive storage not supported by the factory")
+		return false
+	}
+	reader, errReader := archiveFactory.CreateArchiveSpanReader()
+	require.NoError(t, errReader)
+	writer, errWriter := archiveFactory.CreateArchiveSpanWriter()
+	require.NoError(t, errWriter)
```

Comment:
no need to name errors
```suggestion
	reader, err := archiveFactory.CreateArchiveSpanReader()
	require.NoError(t, err)
	writer, err := archiveFactory.CreateArchiveSpanWriter()
	require.NoError(t, err)
```